examples/spring-boot-webflux/src/main/java/example/springframework/boot/webflux/HelloConfiguration.java

```diff
@@ -58,14 +56,18 @@ public ArmeriaClientConfigurator armeriaClientConfigurator() {
             final CircuitBreakerStrategy strategy = CircuitBreakerStrategy.onServerErrorStatus();
             builder.decorator(new CircuitBreakerHttpClientBuilder(strategy).newDecorator());
 
-            // Automatically retry a request when the server returns a 5xx response.
-            builder.decorator(RetryingHttpClient.newDecorator(RetryStrategy.onServerErrorStatus()));
-
-            // Use a custom client factory in order to disable the certificate validation,
-            // which means any certificate received from the server will be accepted.
-            final ClientFactory clientFactory = new ClientFactoryBuilder().sslContextCustomizer(
-                    b -> b.trustManager(InsecureTrustManagerFactory.INSTANCE)).build();
-            builder.factory(clientFactory);
+            // Set a custom client factory.
+            builder.factory(clientFactory());
         };
     }
+
+    /**
+     * Returns a custom client factory which is configured as disabling the certificate validation,
+     * which means any certificate received from the server will be accepted.
+     */
+    @Bean
+    public ClientFactory clientFactory() {
```

Comment:
.. and for readability, we could move this method before `armeriaClientConfigurator()`.